IDEAS/Buildings/Components/ZoneAirModels/AirLeakage.mo

```diff
@@ -84,31 +56,23 @@ equation
     annotation (Line(points={{40,0},{12,0},{12,20}}, color={0,127,255}));
   connect(idealSourceIn.port_b, port_b)
     annotation (Line(points={{60,0},{80,0},{100,0}}, color={0,127,255}));
-  connect(sim.weaBus, weaBus) annotation (Line(
-      points={{-84,92.8},{-40,92.8},{-40,86}},
-      color={255,204,51},
-      thickness=0.5));
-  connect(bou.T_in, weaBus.Te) annotation (Line(points={{6,42},{6,86.05},{-39.95,
-          86.05}}, color={0,0,127}));
 
   connect(reaExpX_air.y, bou.X_in[2])
-    annotation (Line(points={{-19,66},{14,66},{14,42}}, color={0,0,127}));
+    annotation (Line(points={{37,52},{14,52},{14,42}},  color={0,0,127}));
   if Medium.nC>0 then
   connect(bou.C_in[1], weaBus.CEnv)
-    annotation (Line(points={{18,42},{18,86.05},{-39.95,86.05}},
-                                                        color={0,0,127}));
+    annotation (Line(points={{18,42},{18,46},{62,46},{62,86},{10,86},{10,86.05},
+            {-39.95,86.05}},                            color={0,0,127}));
   end if;
   connect(reaPasThr.u, weaBus.X_wEnv)
-    annotation (Line(points={{14,80},{14,86.05},{-39.95,86.05}},
+    annotation (Line(points={{30,82},{30,86.05},{-39.95,86.05}},
                                                         color={0,0,127}));
-  connect(reaPasThr.y, bou.X_in[1]) annotation (Line(points={{14,57},{14,57},{14,
+  connect(reaPasThr.y, bou.X_in[1]) annotation (Line(points={{30,59},{14,59},{14,
           52},{14,42}}, color={0,0,127}));
+  connect(Te.y, bou.T_in)
+    annotation (Line(points={{6,59},{6,56},{6,56},{6,42}}, color={0,0,127}));
     annotation (Icon(coordinateSystem(preserveAspectRatio=false, extent={{-100,
-            -100},{100,100}}),
-                   graphics={Text(
-          extent={{-60,60},{60,-60}},
-          lineColor={0,128,255},
-          textString="ACH")}),                                   Diagram(
+            -100},{100,100}})),                                  Diagram(
         coordinateSystem(preserveAspectRatio=false, extent={{-100,-100},{100,100}})),
     Documentation(revisions="<html>
 <ul>
```

Comment:
revision history is duplicated with partial_AirLeakage. Can this be fixed? I propose you start a new revision history for the partial.